.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: csharp
-mono:
-  - 4.8.1
+sudo: required
+dist: trusty
+dotnet: 1.0.0-preview2-003121
```

Comment:
The only area I am unsure about is the mono part. Is mono required to build net45 on ubunutu? Or can net core build it on travis ( I suspect it can't build it, and that mono needs to build it. If mono builds it can that support netcore csproj files?